### PR TITLE
UI bug fixes 02

### PR DESCRIPTION
**Pull request by Spintacle123**
PR Includes: Fixed UI Issues
 - 1.3.3) Old placeholder image for rewarded weapon
 - 1.3.5) The weapon icon should be aligned with the char icon (same size + same font color 
 - 1.3.6) 'Claim Rewards' not visibly disabled when no rewards available
 - 1.5.1) Weird behaviour with the icon color; Last one is still colored when selecting another one
 - 1.3.7) After a user selects a char, a weapon and then clicks 'Join Raid' he get's a pop-up that the raid has not yet started. We should just visibly disable the 'Join Raid' Button beforehand, remove the pop-up and add a tooltip on hover over the disabled btn that says 'Raid has not yet started. Please wait xx:xx:xx'; Same when no weapon is selected

### CODE REVIEW

File: frontend/src/views/Raid.vue [hunk 1]
```diff
@@ -173,10 +173,16 @@
                       </div>
                   </div>
                   <div class="col-lg-12 join-raid">
-                    <button class="claim-btn" @click="promptRewardClaim()" v-tooltip="'Rewards from Previous Raid'">
+                    <button class="claim-btn" :class="rewardIndexes !== null && rewardIndexes.length > 0 ? 'opacity-1': 'opacity-0'"
+                      @click="promptRewardClaim()" v-tooltip="rewardIndexes !== null && rewardIndexes.length > 0 ?
+                      $t('raid.rewardsFromPrevious'): $t('raid.noAvailableRewards')">
```
Comment (Sellie96) [inline frontend/src/views/Raid.vue:178]: This is a wild bit of logic. Can the logic be moved to a Method and the returned boolean be passed on a :disabled=""  there should be a few examples around, you shouldn't need the css then either

File: frontend/src/views/Raid.vue [hunk 2]
```diff
@@ -173,10 +173,16 @@
...
                       {{$t('raid.claimRewards').toUpperCase()}}
                     </button>
-                    <button v-if="!isMobile()" class="btn-raid"  v-tooltip="$t('raid.joiningCostStamina', {formatStaminaHours})" @click="joinRaidMethod()">
+                    <button v-if="!isMobile()" class="btn-raid"
+                    v-tooltip="!isRaidStarted ? $t('raid.errors.raidNotStarted') :
+                    (!this.selectedWeaponId ? $t('raid.errors.selection') : $t('raid.joiningCostStamina', {formatStaminaHours}))"
+                    @click="joinRaidMethod()"
+                    :class="!isRaidStarted || !this.selectedWeaponId ? 'opacity-0': 'opacity-1'">
```
Comment (Sellie96) [inline frontend/src/views/Raid.vue:185]: I believe the same as above here

File: frontend/src/views/Raid.vue (hunk 1 above)
Comment (Spintacle123) [reply frontend/src/views/Raid.vue:178]: copy

> This is a wild bit of logic. Can the logic be moved to a Method and the returned boolean be passed on a :disabled="" there should be a few examples around, you shouldn't need the css then either

noted

Comment (Spintacle123) [reply frontend/src/views/Raid.vue:178]: > This is a wild bit of logic. Can the logic be moved to a Method and the returned boolean be passed on a :disabled="" there should be a few examples around, you shouldn't need the css then either

I already tried to add method that trigger the disabled but the opacity is not affected..

**Comment by Sellie96** [comment]
1.3.7) Doesn't seem to be complete here. It's still popping the modal with "Raid hasn't started". Think it wants the tooltip replacing and remaining disabled in this case.

File: frontend/src/views/Raid.vue [hunk 3]
```diff
@@ -172,8 +172,10 @@
                         <nft-icon :isDefault="true" :nft="{ type: '5bdust' }"/>
                       </div>
                   </div>
+                  <!-- :class="rewardIndexes !== null && rewardIndexes.length > 0 ? 'opacity-1': 'opacity-0'" -->
```
Comment (Sellie96) [inline frontend/src/views/Raid.vue:175]: Why is this commented?

File: frontend/src/views/Raid.vue [hunk 4]
```diff
@@ -528,6 +530,10 @@ export default Vue.extend({
       Events.$emit('weapon-inventory', true);
     },
 
+    noRewards(){
+      return this.rewardIndexes === null && this.rewardIndexes.length === 0;
```
Comment (Sellie96) [inline frontend/src/views/Raid.vue:534]: There is a typing issue here. Check the build

**Review by Sellie96 — APPROVED** [review]

**Pull request merged**